internal/clientcache/cmd/daemon/start.go

```diff
@@ -319,8 +320,8 @@ func (c *StartCommand) makeBackground(ctx context.Context, dotDir string) (bool,
 	if c.flagRefreshInterval > 0 && c.flagRefreshInterval != daemon.DefaultRefreshInterval {
 		args = append(args, "-refresh-interval", c.flagRefreshInterval.String())
 	}
-	if c.flagFullFetchInterval > 0 && c.flagFullFetchInterval != daemon.DefaultFullFetchInterval {
-		args = append(args, "-full-fetch-interval", c.flagFullFetchInterval.String())
+	if c.flagRecheckSupportInterval > 0 && c.flagRecheckSupportInterval != daemon.DefaultRecheckSupportInterval {
+		args = append(args, "-full-fetch-interval", c.flagRecheckSupportInterval.String())
 	}
```

Comment:
```suggestion
	args = append(args, "-full-fetch-interval", c.flagRecheckSupportInterval.String())
```

Reply:
Since there is a default this is guaranteed to always be populated, any reason to not always pass it in?